crates/matrix-sdk-base/src/client.rs

```diff
@@ -1226,6 +1233,10 @@ impl BaseClient {
             push_rules.notification_power_levels = room_power_levels.notifications;
         }
     }
+
+    pub fn get_ignore_user_list_broadcast_tx(&self) -> &broadcast::Sender<()> {
```

Comment:
You should make this similar to `subscribe_to_unknown_token_errors` in order to avoid exposing the full broadcast. Its users should only be able to receive not also send